### PR TITLE
Add comments about dashboard permission

### PR DESCRIPTION
**Pull request by impactmass**
Update based on work done in on dashboard permissions that went out in [Release 1.2.0](https://github.com/reactioncommerce/reaction/releases/tag/v1.2.0)

### CODE REVIEW

File: developer/architecture/permissions.md [hunk 1]
```diff
@@ -50,6 +50,9 @@ Reaction.registerPackage({
     }]
    });
 ```
+At the point where Packages are published in the app, each registry item permissions are collected and put on the
+package registry [(source)](https://github.com/reactioncommerce/reaction/blob/master/server/publications/collections/packages.js#L31-L56).
+Based on these permissions, restriction to certain functionalities can then be granted or prevented depending on user roles.
```
Comment (zenweasel) [inline developer/architecture/permissions.md]: Not sure how the logic of this sentence works: "restrictions to certain functionalities can be granted". So are we granting restrictions? Maybe something like "Based on these permissions we can enable or disable functionality depending on user roles"?

File: developer/architecture/permissions.md [hunk 2]
```diff
@@ -167,3 +170,25 @@ Helpers in template in templates:
 ```
 
 `/client/modules/core/helpers/permissions.js` exports the `hasPermission` helper.
+
+## Reaction.Apps()
+This core helper method gets all package apps that matches the filter passed in. You can use as in the example below to
+get all enabled packages for payments.
```
Comment (zenweasel) [inline developer/architecture/permissions.md]: "that match"

Comment (zenweasel) [inline developer/architecture/permissions.md]: "You can use this"

File: developer/architecture/permissions.md (hunk 1 above)
Comment (impactmass) [inline developer/architecture/permissions.md]: thanks @zenweasel

**Pull request merged**